cmd/config/subcommand/task_resource_attribute_config.go

```diff
@@ -0,0 +1,72 @@
+package subcommand
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+)
+
+type TaskResourceAttrConfig struct {
```

Comment:
can we avoid creating a special structure? Instead just use the structure in protobuf?

Reply:
is the problem that we cannot support pflags then, lets skip that for now. This way we can support yaml files

Reply:
Done . Now using the protobuf struct of TaskResourceAttr and defined a shadow config on flytectl side.